tests/python/test_spark/test_spark_local.py

```diff
@@ -427,6 +435,12 @@ def setUp(self):
     def get_local_tmp_dir(self):
         return self.tempdir + str(uuid.uuid4())
 
+    def assert_model_compatible(self, model: XGBModel, model_path: str):
+        bst = xgb.Booster()
+        path = glob.glob(f"{model_path}/**/model/part-00000", recursive=True)[0]
+        bst.load_model(path)
+        self.assertEqual(model.get_booster().save_raw("json"), bst.save_raw("json"))
```

Comment:
add a test to assert model file does not include `\n` char.

Reply:
Yeah, per my understanding, seems we don't need to do this, since if there is "\n", the assertion must be failed `self.assertEqual(model.get_booster().save_raw("json"), bst.save_raw("json"))`